src/main/java/mate/academy/intro/config/TestUtil.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.intro.config;
```

Comment:
it should be in test package
```suggestion
package mate.academy.intro.util;
```